src/concourse/pipelines/infrastructure/ovs/static_assets_pipeline.py

```diff
@@ -0,0 +1,171 @@
+                        "REDIS_URL": "redis://localhost:6379/4",
+                        "XDG_CACHE_HOME": "/src/.cache",
+                        "SECRET_KEY": "actions_secret",  # pragma: allowlist secret
+                        "AWS_ACCESS_KEY_ID": "fakeawskey",  # pragma: allowlist secret
+                        "AWS_SECRET_ACCESS_KEY": "fakeawssecret",  # pragma: allowlist secret
+                        "AWS_REGION": "us-east-1",
+                        "CLOUDFRONT_KEY_ID": "cfkeyid",
+                        "DROPBOX_KEY": "foo_dropbox_key",  # pragma: allowlist secret
+                        "ET_PIPELINE_ID": "foo_et_pipeline_id",
+                        "FIELD_ENCRYPTION_KEY": "jtma0CL1QMRLaJgjXNlJh3mtPNcgok0G5ajRCMZ_XNI=",  # pragma: allowlist secret
+                        "GA_DIMENSION_CAMERA": "fake",
+                        "GA_KEYFILE_JSON": "fake",
+                        "GA_VIEW_ID": "fake",
+                        "GA_TRACKING_ID": "fake",
+                        "LECTURE_CAPTURE_USER": "admin",
+                        "MAILGUN_URL": "http://fake_mailgun_url.com",
+                        "MAILGUN_KEY": "foookey",  # pragma: allowlist secret
+                        "ODL_VIDEO_BASE_URL": "http://video.example.com",
+                        "VIDEO_S3_BUCKET": "video-s3",
+                        "VIDEO_S3_TRANSCODE_BUCKET": "video-s3-transcodes",
+                        "VIDEO_S3_THUMBNAIL_BUCKET": "video-s3-thumbs",
+                        "VIDEO_S3_SUBTITLE_BUCKET": "video-s3-subtitles",
+                        "VIDEO_S3_WATCH_BUCKET": "video-s3-watch",
+                        "VIDEO_CLOUDFRONT_DIST": "video-cf",
+                        "YT_ACCESS_TOKEN": "fake",
+                        "YT_REFRESH_TOKEN": "fake",
+                        "YT_CLIENT_ID": "fake",
+                        "YT_CLIENT_SECRET": "false",  # pragma: allowlist secret
+                        "YT_PROJECT_ID": "fake",
+                    },
+                    run=Command(
+                        path="/bin/entrypoint.sh",
+                        args=[
+                            "bash",
+                            "-ceux",
+                            textwrap.dedent(
+                                """mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd
+                                 cd odl-video-service
+                                 chmod 777 .
+                                 env > .env
+                                 docker-compose build
+                                 whoami
+                                 docker-compose run -u root watch yarn install --frozen-lockfile --ignore-engines --prefer-offline
+                                 docker-compose run -u root watch node node_modules/webpack/bin/webpack.js --config webpack.config.prod.js --bail
+                                 docker-compose run web python manage.py collectstatic --no-input
```

Comment:
Do we need to use a full docker-compose stack for running the `collectstatic` step? Can we run that in just the OVS container?

Reply:
That is a good question. I figured since we were already here with a good amount of the rest of the stack for the node steps, we could just bang out the `collectstatic` as well. I will look into it further.